libs/kml/serdes_geojson.cpp

```diff
@@ -264,4 +285,132 @@ void DeserializerGeoJson::Deserialize(std::string_view content)
   }
 }
 
+std::vector<std::vector<double>> convertPoints2GeoJsonCoords(std::vector<geometry::PointWithAltitude> points)
+{
+  std::vector<std::vector<double>> pairs;
+  pairs.resize(points.size());
+  for (size_t i = 0; i < points.size(); i++)
+  {
+    auto const & p = points[i];
+    auto const [lat, lon] = mercator::ToLatLon(p);
+    pairs[i] = std::vector<double>{lon, lat};
+  }
+  return pairs;
```

Comment:
What about the altitude? Add a bool flag when it should be serialized?

```suggestion
  pairs.reserve(points.size());
  for (auto const & point : points)
  {
    auto const [lat, lon] = mercator::ToLatLon(point);
    pairs.emplace_back(lon, lat);
  }
  return pairs;
```